Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme

```diff
@@ -256,6 +256,15 @@
          <TestPlanReference
             reference = "container:Tests/Smoketest3.xctestplan">
          </TestPlanReference>
+         <TestPlanReference
+            reference = "container:Tests/Fennec_Enterprise_XCUITests.xctestplan">
```

Comment:
I would also rename this test plan to Full Functional or something similar because we are removing that scheme name and it does not make much sense anyway